core/iwasm/aot/aot_runtime.c

```diff
@@ -5020,3 +5020,121 @@ aot_get_module_name(AOTModule *module)
 {
     return module->name;
 }
+
+void
+aot_resolve_symbols(AOTModule *module)
```

Comment:
Is it better to change the return type to bool for aot_resolve_symbols, aot_resolve_import_func, wasm_resolve_symbols, wasm_resolve_import_func and wasm_runtime_resolve_symbols? So that developer can know whether wasm_runtime_resolve_symbols is successful or not?

And this function we try linking all the functions (don't stop when failed to link a function), and eventually return false if a function is linked unsuccessfully?

Reply:
Agree on it returning bool. That seems like good information. Failure to link a function is treated as a possibly benign error when not using no_resolve and is indeed sometimes not a problem so I think it's best to keep linking as many functions as possible.

Reply:
That seems better. Will fix.